src/solvers/sat/satcheck_minisat2.cpp

```diff
@@ -175,13 +175,15 @@ void satcheck_minisat2_baset<T>::lcnf(const bvt &bv)
 
 #ifndef _WIN32
 
+/*
```

Comment:
What's going on here? Dead-code cleanup? If so, great, but let's have it's own PR for it so it gets accepted.